app/assets/main/react_components/footprint_form/Container.jsx

```diff
@@ -4,15 +4,20 @@ import FootprintForm from './FootprintForm.jsx';
 const Container = ({ footprint, calculator, lifestyleFootprintsText, lang }) => {
 
   return (
-      <div className="callout">
-          <FootprintForm 
-            questions={JSON.parse(lifestyleFootprintsText).lifestyle_footprints.questions} 
-            options={JSON.parse(lifestyleFootprintsText).lifestyle_footprints.options} 
-            footprint={JSON.parse(footprint)} 
-            calculator={JSON.parse(calculator)}
-          />
+      <div className="space-y-6"> 
+        <div className="callout">
+            <FootprintForm 
+              questions={JSON.parse(lifestyleFootprintsText).lifestyle_footprints.questions} 
+              options={JSON.parse(lifestyleFootprintsText).lifestyle_footprints.options} 
+              footprint={JSON.parse(footprint)} 
+              calculator={JSON.parse(calculator)}
+            />
+        </div>
+          <div className="text-sm">
+            <a className="link" href="https://www.goclimate.com/blog/methodology-behind-the-carbon-footprint-calculator/" target="_blank">{JSON.parse(lifestyleFootprintsText).lifestyle_footprints.methodology}</a>
+          </div>
```

Comment:
Lite tokig indentering verkar ha letat sig in!

```suggestion
        </div>
        <div className="text-sm">
          <a className="link" href="https://www.goclimate.com/blog/methodology-behind-the-carbon-footprint-calculator/" target="_blank">{JSON.parse(lifestyleFootprintsText).lifestyle_footprints.methodology}</a>
        </div>
```